spec/beautify-languages-spec.coffee

```diff
@@ -44,12 +44,11 @@ describe "BeautifyLanguages", ->
 
   # Activate all of the languages
   allLanguages = [
-    "c", "clojure", "coffee-script", "css", "d", "html",
-    "java", "javascript", "json", "less",
-    "mustache", "objective-c", "perl", "php",
-    "python", "ruby", "sass", "sql", "svg",
-    "xml", "csharp", "gfm", "marko",
-    "go", "html-swig", "lua"
+    "c", "clojure", "coffee-script", "css", "csharp", "d", "fuse",
+    "gfm", "go", "html", "html-swig", "java", "javascript",
+    "json", "less", "lua", "marko", "mustache", "objective-c",
+    "perl", "php", "python", "ruby", "sass", "sql",
+    "svg", "xml"
```

Comment:
I forgot, it also needed to be added to Travis CI's install list: https://github.com/prettydiff/atom-beautify/blob/002b668ca23b2a0932ee3cd079c212c6f5e731d9/.travis.yml#L59 
And AppVeyor: https://github.com/prettydiff/atom-beautify/blob/002b668ca23b2a0932ee3cd079c212c6f5e731d9/appveyor.yml#L136

Looks like `language-fuse` is missing.